rj_gameplay/rj_gameplay/play/test_motion_planning.py

```diff
@@ -61,7 +61,7 @@ def tick(
             self.move_skill = move.Move(
                 robot=self.robot,
                 target_point=self.target_point,
-                face_point=self.face_point,
+                # face_point=self.face_point,
```

Comment:
```suggestion
                face_point=self.face_point,
```
this explains why face point wasn't working yesterday...